src/hierarchies/likelihoods/states/fa_state.h

```diff
@@ -12,6 +12,17 @@
 
 namespace State {
```

Comment:
I'd add one line of doc to `compute_wood_factors()` that points to the documentation of `bayesmix::compute_wood_chol_and_logdet()`